common/presenters/allocations-to-table.js

```diff
@@ -5,33 +5,43 @@ const tablePresenters = require('./table')
 const tableConfig = [
   {
     head: 'allocations::move_size',
-    attributes: {
+    row: {
+      text: 'moves_count',
       attributes: {
         scope: 'row',
       },
     },
-    row: 'moves_count',
   },
   {
     head: 'allocations::requested',
-    row: data => filters.formatDate(data.created_at),
+    row: {
+      text: data => filters.formatDate(data.created_at),
+    },
   },
   {
     head: 'allocations::move_from',
-    row: 'from_location.title',
+    row: {
+      text: 'from_location.title',
```

Comment:
Why all the `htmls`s to 'text`s?

Reply:
They were all made `html` originally but that shouldn't be the default. Using `html` in the [Table component](https://design-system.service.gov.uk/components/table/) will escape the input. We should only use `html` if we need to and can ensure there is no unescaped user inputted data in there.